### PR TITLE
render all blob contributions in one parallel webgl canvas

### PR DESCRIPTION
**Pull request by alejandrogarciasalas**
uses https://github.com/FarazzShaikh/react-three-scissor to render all the blobs in a parallel webgl canvas. The library ports https://threejs.org/manual/#en/multiple-scenes to react-three-fiber , I was going to write something like this myself when I bumped into this library, it was published ~2 months ago, took a look at the code and seems good enough. The solution essentially renders a parallel webgl canvas `ScissorCanvas`, looks for all `ScissorWindow`s gets their position using getBoundigRect and render's the blob mesh on that same position on the canvas.

Took a look at this site https://shapefarm.net/ and they effectively do the same too, except they pass some data to the renderer by `data-` attribute.
Screen Shot 2022-01-22 at 9 26 12 PM

@spencerc99 I still need to do a bit more thorough testing that contributions render fine in all contributions card, but almost there.

### CODE REVIEW

File: server/common/server-api/index.ts [hunk 1]
```diff
@@ -1,103 +0,0 @@
-// import type { Prompt, Pattern } from "@prisma/client";
```
Comment (spencerc99) [inline server/common/server-api/index.ts:1]: can you revert this

Comment (alejandrogarciasalas) [reply server/common/server-api/index.ts:1]: aah oops

**Comment by spencerc99** [comment]
ok here goes nothing ✨
https://discord.com/channels/906285374572224562/925465611025395742/936523719902363738

File: browser/src/components/react-three-scissor/store.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
+import create from "zustand";
```
Comment (spencerc99) [inline browser/src/components/react-three-scissor/store.tsx:1]: ideally we burn this code and redo it lol

**Pull request merged**